README.md

```diff
@@ -12,10 +12,11 @@ I'd be happy to help if you'd like to make a version of this for your own city.
 Here are the steps to follow if you want to run COVID-19 MTL locally for testing or development purposes:
 
 1. Clone this repository then navigate to the checkout
-2. Create a virtual environment: `python3.8 -m venv .venv`
-3. Activate the environment: `. .venv/bin/activate` (the command is `.`, meaning source or include)
-4. Install dependencies: `pip install -r requirements.txt`
-5. Run the app!  `python -m app`
+1. Create a virtual environment: `python3.8 -m venv .venv`
+1. Activate the environment: `. .venv/bin/activate` (the command is `.`, meaning source or include)
+1. Install dependencies: `pip install -r requirements.txt`
+1. For development: Install development dependencies `pip install -r requirements-dev.txt`
+1. Run the app!  `python -m app`
```

Comment:
Is this change required?  It makes the source text less readable in my opinion, which is too bad for a plain-text format.

Reply:
Thanks for your feedback. I am planning to make some more contributions so I thought it might make sense to add that. But I'm fine if there's no interest.

> Is this change required? It makes the source text less readable in my opinion, which is too bad for a plain-text format.

Ideally, if there's dependencies for different environments, a single command would be nice. Unfortunately, `pip-tools` does not seem to add `-r requirements.txt` at the top of `requirements-dev.txt`.

Reply:
I was only asking about the change in the numbered list here!

Reply:
Oh, sorry! Good point, was only thinking about how it will be rendered. I reverted that and made the dependency installation a one-liner.